content/80.troubleshooting.md

```diff
@@ -13,7 +13,7 @@ functionalities, here are detailed solutions and best practices to ensure a smoo
 If you encounter issues using the JavaScript SDK during the installation, refer to the troubleshooting steps:
 
 1. **Network selection**: Ensure you are using the Layer 2 (L2) network instead of Layer 1 (L1) Sepolia. The
-correct endpoint for the L2 network is: [ZKsync Sepolia Explorer](%%zk_testnet_block_explorer_url%%/).
+correct endpoint for the L2 network is: [ZKsync Sepolia Explorer](https://sepolia.era.zksync.dev).
```

Comment:
Would you mind changing this to `%%zk_testnet_rpc_url%%`? Unfortunately I don't have anywhere these tags exist for one to find and use easily, sorry about that. I'll try and make these tags more public. Here's an example of some of the existing tags being used in a partial in the ZKsync Docs: https://github.com/matter-labs/zksync-docs/blob/main/content/_partials/_testnet-network-details.md

Reply:
Oh, alright. I'll make the change.
Good to know a tag exists for it. Perhaps we could add a page under the Contributing section of the main docs to list these tags and others. It would be like a page of URL constants, of sorts.